src/apps/chifra/internal/scrape/pause.go

```diff
@@ -7,21 +7,32 @@ import (
 	"github.com/TrueBlocks/trueblocks-core/src/apps/chifra/pkg/rpcClient"
 )
 
-func (opts *ScrapeOptions) Pause(progressThen *rpcClient.MetaData) {
-	// we always pause at least a quarter of a second to allow the node to 'rest'
-	time.Sleep(250 * time.Millisecond)
+const (
+	minimumSleep = time.Second / 4
+)
+
+func (opts *ScrapeOptions) Pause(progressThen *rpcClient.MetaData) <-chan time.Time {
```

Comment:
Thanks for fixing this. This Pause code has always been a bit of troubling. Can you add a comment here explaining (briefly) its operation?